.travis.yml

```diff
@@ -1,29 +1,10 @@
-# Config file for automatic testing at travis-ci.org
-# This file will be regenerated if you run travis_pypi_setup.py
-
 language: python
 python:
-  - 3.5
-  - 3.4
-  - 3.3
-  - 2.7
-  - 2.6
-
-# command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
-install: pip install -U tox-travis
-
-# command to run tests, e.g. python setup.py test
-script: tox
-
-# After you create the Github repo and add it to Travis, run the
-# travis_pypi_setup.py script to finish PyPI deployment setup
-deploy:
-  provider: pypi
-  distributions: sdist bdist_wheel
-  user: jonbgallant
-  password:
-    secure: PLEASE_REPLACE_ME
-  on:
-    tags: true
-    repo: azure/iotedgedev
-    python: 2.7
+  - "3.6"
+  - "3.5"
+  - "3.4"
+  - "2.7"
+install:
+  - pip install -r requirements_travis.txt
+script:
+  - pytest -m unit # & pylint iotedgedev # or py.test for Python versions 3.5 and below
```

Comment:
pylint runs successfully, but I'm not including it in the check yet because we have [significant lint failures](https://travis-ci.org/Azure/iotedgedev/jobs/398015885) at the moment.